app/tests/spec/views/form.js

```diff
@@ -336,11 +336,15 @@ describe('views/form', function () {
   });
 
   describe('showValidationError', function () {
-    it('creates a tooltip', function () {
-      view.on('validation_error', function (done) {
```

Comment:
This test was wrong before, `done` wasn't a function but the error wasn't being caught because this is inside an event handler. `done` should be on the `it`.